src/styles/block/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $main-borderCard-color;
+  border-radius: 5px;
+  background-color: $main-backgroundCard-color;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    color: $main-textCard-color;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    color: $secondary-textCard-color;
+  }
+
+  &__feedback-container {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__feedback-reviews {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    height: 16px;
+    text-align: right;
+    color: $main-textCard-color;
+    margin: 0;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-name {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 150%;
+    color: $secondary-textCard-color;
+    margin: 0;
+  }
+
+  &__price-dollars {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: $main-textCard-color;
+    margin: 0;
+  }
+
+  &__link-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    box-sizing: border-box;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for an element with content inside. use padding instead